docs: link custom validation rules & add custom rules for login (#363)
codeigniter4/shield: docs/customization.md

        ],
    ];
```

### Login

> Similar to what was explained about the setting of validation rules for **Registration** section, you can add the rules of the login form in file (**app/Config/Validation.php**) and change the rules.

Reviewer: ```suggestion
Similar to the process for validation rules in the **Registration** section, you can add rules for the login form to **app/Config/Validation.php** and change the rules.
```
